### PR TITLE
chore(localized-rich-text-input): show expand button

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

As requested by design, we should show the `expand` button when there is more than 32px of collapsed text.

### CODE REVIEW

File: src/components/inputs/localized-rich-text-input/editor.js [hunk 1]
```diff
@@ -146,14 +146,24 @@ const Editor = props => {
...
                   <LeftColumn />
                   <RightColumn>
                     <FlatButton
                       onClick={toggle}
-                      label={intl.formatMessage(messages.collapse)}
-                      icon={<AngleUpIcon size="small" />}
+                      label={
+                        isOpen
+                          ? intl.formatMessage(messages.collapse)
+                          : intl.formatMessage(messages.expand)
```
Comment (tdeekens) [inline src/components/inputs/localized-rich-text-input/editor.js]: Nit: `intl.formatMessage(isOpen ? messages.collapse : messages.expand)`

File: src/components/inputs/localized-rich-text-input/editor.js [hunk 2]
```diff
@@ -146,14 +146,24 @@ const Editor = props => {
...
+                      }
+                      icon={
+                        isOpen ? (
```
Comment (tdeekens) [inline src/components/inputs/localized-rich-text-input/editor.js]: Sometimes I wonder if doing a 

```js
const ToggleButtonIcon = props => props.collapsed ? <AngleDownIcon size="small" /> : <AngleUpIcon size="small" />; 
```

and then

```js
icon={<ToggleButtonIcon collapsed={!props.isOpen} />}
```

is just overly eager or now. 🤷‍♂

File: src/components/inputs/localized-rich-text-input/rich-text-input.js [hunk 3]
```diff
@@ -113,15 +113,7 @@ class RichTextInput extends React.PureComponent {
       return (
         <LanguagesControlButton
           isClosed={true}
-          onClick={() => {
-            // expand all multiline language inputs in case the
```
Comment (montezume) [inline src/components/inputs/localized-rich-text-input/rich-text-input.js]: neither Filip or Maria could remember why we had this special logic...

Comment (jonnybel) [reply src/components/inputs/localized-rich-text-input/rich-text-input.js]: This logic is also in `<LocalizedMultilineTextInput>`. Should the behaviour remain the same?

Comment (montezume) [reply src/components/inputs/localized-rich-text-input/rich-text-input.js]: I'll ask Filip

File: src/components/inputs/localized-rich-text-input/localized-rich-text-input.visualroute.js [hunk 4]
```diff
@@ -61,7 +61,11 @@ const DefaultRoute = () => (
     <Spec label="minimal" omitPropsList>
       <LocalizedRichTextInput
         onChange={() => {}}
-        value={initialValue}
+        value={{
+          en: emptyValue,
```
Comment (montezume) [inline src/components/inputs/localized-rich-text-input/localized-rich-text-input.visualroute.js]: I wanted a VRT that showed an input with one line

File: src/components/inputs/localized-rich-text-input/rich-text-input.js (hunk 3 above)
Comment (montezume) [reply src/components/inputs/localized-rich-text-input/rich-text-input.js]: Updated in last commit

**Pull request merged**